integration-tests/src/ethereum/ctc/contracts/GluwaCreditVestingToken.sol/CreditcoinBase.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
+  "_format": "hh-sol-dbg-1",
+  "buildInfo": "../../../build-info/dad11b178307e638d0c911178d0e0d94.json"
```

Comment:
I think these `.dbg` files are not needed and it should be safe to exclude them. The referenced path isn't available in this commit anyway.

Reply:
this is probably true, i just copied the entire artifacts folder over